docs/developer-guide/plugins.md

```diff
@@ -163,6 +163,30 @@ In addition to the standard parsers mentioned in the ["Working on rules"](rules.
 
 Have a look through [stylelint's internal utils](https://github.com/stylelint/stylelint/tree/master/lib/utils) and if you come across one that you need in your plugin, then please consider helping us extract it out into an external module.
 
+## Peer dependency
+
+To make clear that your plugin requires which versions of stylelint, you have to declare stylelint as a `peerDependency` (**NOT** `dependency`) in your `package.json`. Because, if a user would try to install plugins declared as a `dependency` which each require different versions of stylelint, different versions of stylelint would be installed **unexpectedly** at same time.
```

Comment:
Let's go with the following:

---

## Peer dependencies

You should express, within the `peerDependency` key (and **not** within the `dependency` key) of your plugin's `package.json`, what version(s) of stylelint your plugin can be used with. This is to ensure that different versions of stylelint are not unexpectedly installed.

For example, to express that your plugin can be used with stylelint versions 7 and 8:

```json
{
  "peerDependencies": {
    "stylelint": "^7.0.0 || ^8.0.0"
  }
}
```
---

As much as I love emojis, not using them is more inline with the current tone of the documentation. Also, I've tried to make it a little terser as our documentation is growing a lot.

Reply:
@jeddy3 Thank you for review!

Certainly, the use of emojis in this case seems a little strange 😓 

Also, I feel my documentation is roundabout, so I agree you proposal which is more clear and easier to understand 👍